### PR TITLE
Qss update3

### PR DESCRIPTION
**Pull request by towerwatchman**

### CODE REVIEW

File: .github/workflows/prerelease.yml
Comment (KJNeko) [inline .github/workflows/prerelease.yml:1]: This probably needs to be removed as it seems mildly broken. It sat around for 14 hours waiting on a runner.

File: atlas/core/database/record/GameBannerLoading.cpp [hunk 1]
```diff
@@ -212,7 +217,10 @@ namespace atlas::records
 			}
 
 			if ( promise.isCanceled() ) return;
-			//pixmap = pixmap.scaled( size, Qt::AspectRatioMode( scale_type ) );
+			//Temp fix for image fill
+			spdlog::
+				info( "target_size h:{}, w:{}, scale_type:{}", target_size.height(), target_size.width(), scale_type );
```
Comment (KJNeko) [inline atlas/core/database/record/GameBannerLoading.cpp:222]: Your going to give me an anyuerism.

File: .github/workflows/prerelease.yml
Comment (KJNeko) [reply .github/workflows/prerelease.yml:1]: Small adendum to this. Currently PR #72 has things for this but it's just waiting on testing to be done. For now though we can probably fix it up to ensure it works then merge it in and figure out testing later.

File: atlas/core/database/record/GameBannerLoading.cpp (hunk 1 above)
Comment (towerwatchman) [reply atlas/core/database/record/GameBannerLoading.cpp:222]: Yeah that should have been deleted.

File: .github/workflows/prerelease.yml
Comment (towerwatchman) [reply .github/workflows/prerelease.yml:1]: ohhhhhh, this makes way more sense why i was running now. Ill delete this

**Pull request merged**